source/10-compliance/03-counter-reporting-for-consortia.rst

```diff
@@ -7,38 +7,52 @@
 COUNTER Reporting for Consortia
 -------------------------------
 
-Consortia license content for their members and consortium administrators need access to COUNTER statistics that show how each member has used the licensed resources.
+Consortia license content for their members, and consortia managers and administrators need access to COUNTER statistics that show how each member within a given consortium has used the licensed resources.
 
 
-Access to SUSHI Credentials for Member Sites
-""""""""""""""""""""""""""""""""""""""""""""
+Types of Consortia-Level Report
+"""""""""""""""""""""""""""""""
 
-Content providers MUST support the /members COUNTER_SUSHI API path to provide the consortium with the list of their members on the platform and the SUSHI credentials for each member. This will enable tools to be created to efficiently retrieve member usage and create separate or consolidated reporting.
+There are three types of consortia-level reporting:
 
+* Report providers MUST offer summary reports that show total usage for all members of the consortium rolled up to the consortia level, and which cannot be broken down by institution. The totals on the summary report may differ from the sum of the totals on individual member reports for the same items if an authentication method used identifies to multiple member sites and usage it attributed to each such site (e.g. overlapping IP ranges).
+* Report providers may elect to offer detailed reports that show total usage for all members of the consortium rolled up to the consortia level, and which can be broken down by institution. As with the summary report, the totals on the detailed report may differ from the sum of the totals on individual member reports for the same items.
+* Institutional reports are the individual institutional-level reports. Consortia managers and administrators MUST be able to retrieve institutional reports.
 
-Privacy and Confidentiality
-"""""""""""""""""""""""""""
+Note that consortia-level reports are restricted to COUNTER Reports. The Standard Views of the COUNTER Reports are not available as consortia-level reports.
```

Comment:
The Standard Views are required in 5.0 for the summary reports. Why would we restrict this to the COUNTER Reports in 5.1?